SFUsat/sfu_cmds.c

```diff
@@ -23,7 +23,7 @@ char buffer[250];
 int8_t cmdGet(const CMD_t *cmd) {
 	switch (cmd->subcmd_get_id) {
 		case CMD_GET_TASKS: {
-		    serialSend("Task\t\tState\tPrio\tStack\tNum\n");
+			serialSend("Task\t\tState\tPrio\tStack\tNum\n");
```

Comment:
Just a style thing, but next time if `git diff` shows whitespace changes like this, it's better to revert these changes so the commit contains less noise.

And regarding tabs/spaces: I prefer tabs too, but I think Halcogen code generation uses spaces, so that's why the project uses spaces (I think that was my reasoning :o). I also thought I told CCS to use spaces for formatting; it seems that isn't working as intended. We should figure out what to use and stick to it moving forward.

Reply:
Ah shoot didn't realize that. This was from an auto format if I remember right.